compiler/il/OMRResolvedMethodSymbol.hpp

```diff
@@ -252,7 +252,9 @@ class OMR_EXTENSIBLE ResolvedMethodSymbol : public TR::MethodSymbol
    bool hasCheckcastsOrInstanceOfs()         { return _methodFlags2.testAny(HasCheckCasts|HasInstanceOfs); }
    bool hasBranches()                        { return _methodFlags2.testAny(HasBranches); }
    void setHasBranches(bool b)               { _methodFlags2.set(HasBranches, b); }
-
+   bool hasVectorAPI()                       { return _methodFlags2.testAny(HasVectorAPI); }
+   void setHasVectorAPI(bool b)              { _methodFlags2.set(HasVectorAPI, b); }
```

Comment:
Will the code to call this `setHasVectorAPI` routine be in downstream projects ? Note that this property would need to be propagated through inlining as well, i.e. the caller would need to inherit the callee's property.

Reply:
Right, this is just to introduce the flag to be used later.

Reply:
Okay thanks, as long as you do plan to introduce code later to propagate the property through inlining either in OMR itself or in downstream projects, I am good with this change. Approving now.

Reply:
I also presume that you will be adding the disable option mentioned in the TODO comment when you actually start setting/using this flag properly ?

Reply:
Right. I will check if VectorAPIExpansion is disabled. Unfortunately, can't introduce the option until the opt itself is merged.